Tasks/DownloadBuildArtifactsV0/task.json

```diff
@@ -84,6 +84,15 @@
                 "specific": "Specific version"
             }
         },
+        {
+            "name": "partiallySucceededBuilds",
```

Comment:
Should we rename this to allowPartiallySucceededBuilds and other places wherever we consume the same?

Reply:
Thanks for the review. I just renamed the parameter as requested.